xrpl/models/transactions/escrow_create.py

```diff
@@ -2,7 +2,10 @@
 from __future__ import annotations  # Requires Python 3.7+
 
 from dataclasses import dataclass, field
-from typing import Dict, Optional
+from typing import Dict, Optional, TypedDict
+
+# CK: TODO Find a py.typed or library stub for cryptoconditions
+from cryptoconditions import PreimageSha256  # type: ignore
```

Comment:
Please let me know if you have any ideas to get `mypy` to work with a cryptographic library